ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.14.yaml

```diff
@@ -286,6 +286,24 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-vsphere-ipi-customized-resource
+- as: vsphere-ipi-disktype-eagerzeroedthick-f28
+  cron: 46 6 29 * *
+  steps:
+    cluster_profile: vsphere-connected-2
```

Comment:
I think we should `vsphere-elastic` for now.

Reply:
got it.

Reply:
one question. i saw vsphere-connected-2 still in use. the deprecated one was vsphere-2.
are we going to get all connected job move to vsphere-elastic?

Reply:
Right.

Reply:
ok